### PR TITLE
WalletCore upgrade + blast + Cronos

### PR DESCRIPTION
**Pull request by enriquesouza**
I've upgraded Wallet Core so it can have Blast.

Added Blast

Sent ETH on Blast OK

Added Cronos

Send CRO on Cronos OK

### CODE REVIEW

File: voltixApp/VoltixApp/Stores/TokensStore.swift [hunk 1]
```diff
@@ -65,6 +65,12 @@ class TokensStore {
         
         Coin(chain: Chain.polygon, ticker: "WETH", logo: "wETH", address: "", priceRate: 0.0, chainType: ChainType.EVM, decimals: "18", hexPublicKey: "", feeUnit: "Gwei", priceProviderId: "ethereum", contractAddress: "0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619", rawBalance: "0", isNativeToken: false, feeDefault: "120000"),
         
+        Coin(chain: Chain.blast, ticker: "ETH", logo: "eth", address: "", priceRate: 0.0, chainType: ChainType.EVM, decimals: "18", hexPublicKey: "", feeUnit: "Gwei", priceProviderId: "ethereum", contractAddress: "", rawBalance: "0", isNativeToken: true, feeDefault: "21000"),
```
Comment (johnnyluo) [inline voltixApp/VoltixApp/Stores/TokensStore.swift:68]: Can you please update the `feeDefault` to `23000` , for swap , it need to cater for memo, which will use more gas

**Review by johnnyluo — CHANGES_REQUESTED** [review]
one small change

File: voltixApp/VoltixApp/Stores/TokensStore.swift (hunk 1 above)
Comment (enriquesouza) [reply voltixApp/VoltixApp/Stores/TokensStore.swift:68]: Done

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**